### ISSUE TITLE
EmberJS issue

### ISSUE THREAD

**Issue by encore1908** [MindscapeHQ/raygun4js#101]
Hi,

I'm using Ember JS, and i would love to use Raygun for all errors handling... But, now i cannot send any errors, cause every time when i do :
```javascript
Raygun.send(error);
```
I'm getting same error in console:
```javascript
Uncaught SyntaxError: Invalid regular expression: /function superFunction(){.....}/;
```
I believe that problem with lines:
```javascript
navigator.userAgent.match(_excludedUserAgents[userAgentIndex] 
```
and 
```javascript
window.location.hostname && window.location.hostname.match(_excludedHostnames[hostIndex])
```

**Comment by fundead** [comment]
Hi,

Just looking into this one now for you. Have you set either of the excluded hostname or user agents?

Regards,

Callum Gavin
Mindscape Limited

**Comment by encore1908** [comment]
No, i didn't :(

**Comment by fundead** [comment]
Think I might have a solution, would you mind grabbing one of the artifacts from here and confirming it fixes the issue - https://github.com/MindscapeHQ/raygun4js/tree/fix-exclude-default-bug/dist

**Comment by encore1908** [comment]
Nice! Everything works! 
Thank you!

**Comment by fundead** [comment]
Great to hear, this has been released as v1.16.1 if you want to grab it from the package managers.
